LinqChallengesFivePack/LinqChallengesFivePack/LinqChallenges.cs

```diff
@@ -27,15 +29,25 @@ public List<Player> PlayerAgeSort()
         {
             var players = "Jeff Prosise, 04/01/1986; Jos Sagan, 04/22/1983; Mariah Davis, 09/08/1985; Ally Shaw, 12/22/1995; Hector Ramirez, 02/12/1991; James Hansen, 10/05/1983";
            
-            return new List<Player>();
+            return players.Split(";")
+                .Select(allParts => allParts.Split(","))
+                .Select(parts => new Player
+                    {
+                        Name = parts[0].Trim(),
+                        Birthday = DateTime.Parse(parts[1].Trim()),
+                    })
+                .OrderBy(p => p.Birthday)
+                .ToList();
         }
 
         //Calculate how long the album is, in seconds, given track lengths
         public double CalculateAlbumDurationSeconds()
         {
             string albumTrackLengths = "4:12,2:43,3:51,4:29,3:24,3:14,4:46,3:25,4:52,3:27";
             
-            return 0;
+            return albumTrackLengths.Split(",")
+                .Select(track => TimeSpan.ParseExact(track, "m':'ss", null).TotalSeconds)
```

Comment:
I didn't even think to parse these as a `TimeSpan` 🤣. So simple. Great work. Don't look at mine 😅.